pages/docs/reference/whatsnew14.md

```diff
@@ -574,3 +574,306 @@ Complete the initial configuration, and when you add a new dependency to `cocoap
+        }
+    
+        val emptyList = emptyList<Int>()
+        emptyList.reduceIndexedOrNull {index, a, b -> index + a + b} // null
+    //sampleEnd
+    }
+    ```
+    </div>
+
+* `runningFold()` and `runningReduce()` are introduced as synonyms for [`scan()`](/api/latest/jvm/stdlib/kotlin.collections/scan.html)
+and [`scanReduce()`](/api/latest/jvm/stdlib/kotlin.collections/scan-reduce.html). Such names are more consistent with the
+related functions `fold()` and `reduce()`.
+
+    <div class="sample" markdown="1" theme="idea" data-min-compiler-version="1.4">
+    
+    ```kotlin
+    fun main() {
+    //sampleStart
+        val numbers = mutableListOf(0, 1, 2, 3, 4, 5)
+        val runningReduceSum = numbers.runningReduce() { sum, item -> sum + item} // [0, 1, 3, 6, 10, 15]
+        val runningFoldSum = numbers.runningFold(10) { sum, item -> sum + item} // [10, 10, 11, 13, 16, 20, 25]
+    //sampleEnd
+    }
+    ```
+    </div>
+
+* `sumOf()` takes a selector function and returns a sum of its values on all elements of a collection.
+`sumOf()` can produce sums of the types `Int`, `Long`, `Double`, `UInt`, `ULong`. On the JVM, `BigInteger` and `BigDecimal` are also available.
+
+    <div class="sample" markdown="1" theme="idea" data-min-compiler-version="1.4">
+    
+    ```kotlin
+    data class OrderItem(val name: String, val price: Double, val count: Int)
+    
+    fun main() {
+    //sampleStart
+        val order = listOf<OrderItem>(
+            OrderItem("Cake", price = 10.0, count = 1),
+            OrderItem("Coffee", price = 2.5, count = 3),
+            OrderItem("Tea", price = 1.5, count = 2))
+    
+        val total = order.sumOf { it.price * it.count} // Double
+        val count = order.sumOf { it.count } // Int
+    //sampleEnd
+        println("You've ordered $count items that cost $total in total")
+    }
+    ```
+    </div>
+
+* `minOf()` and `maxOf()` functions return the minimum and the maximum value of the given selector function on the collection items.
+
+    <div class="sample" markdown="1" theme="idea" data-min-compiler-version="1.4">
+    
+    ```kotlin
+    data class OrderItem(val name: String, val price: Double, val count: Int)
+    
+    fun main() {
+    //sampleStart
+        val order = listOf<OrderItem>(
+            OrderItem("Cake", price = 10.0, count = 1),
+            OrderItem("Coffee", price = 2.5, count = 3),
+            OrderItem("Tea", price = 1.5, count = 2))
+        val highestPrice = order.maxOf { it.price }
+    //sampleEnd
+        println("The most expensive item in the order costs $highestPrice")
+    }
+    ```
+    </div>
+
+    There are also `minOfWith()` and `maxOfWith()`, which take a `Comparator` as an argument, and `*OrNull()` versions
+of all four functions that return `null` on empty collections.
+
+    In turn, `min()` and `max()` functions have been renamed to `minOrNull()` and `maxOrNull()` to comply with the naming
```

Comment:
Same as in 1.4-M3 blog post, it makes sense to tell about minOrNull/maxOrNull first, and then to introduce minOf/minOfOrNull functions.